File: cpp/28c_1676A_Lucky.cpp
#include<bits/stdc++.h>

int main(){
    int t, n, m;
    std::vector<std::string> ans;
    std::string s;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>s;
        n = s[0] + s[1] + s[2];
        m = s[3] + s[4] + s[5];
        if (n == m){
            ans.push_back("YES");
        }
        else{
            ans.push_back("NO");
        }
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/28c_1676A_Lucky.py
def main():
    t = int(input())
    ans = []
    for _ in range(t):
        s = input().strip()
        n = int(s[0]) + int(s[1]) + int(s[2])
        m = int(s[3]) + int(s[4]) + int(s[5])
        if n == m:
            ans.append("YES")
        else:
            ans.append("NO")
    for i in ans:
        print(i)

if __name__ =="__main__":
    main()